tests/test_assets/model_details/input/model_details_allele_info_good_test_input.csv

```diff
@@ -0,0 +1,6 @@
+model,gene,mgi_gene_id,gene_ensembl_id,allele,allele_type,mgi_allele_id
```

Comment:
Can you add a test file for missing data for this data set? Unless there's already something missing in this file, hard to tell.